content/cli/orch_cli/secrets.md

```diff
@@ -36,17 +36,19 @@ One of these flags:
 
 #### Optional flags:
 
-* `-u, --update-if-exists` - Update secret with new value if it already exists
+* `-u, --update-if-exists` - Update secret value if secret key already exists. [This option is deprecated; use cfy secrets update command instead]. You cannot use this argument with arguments: [visibility, hidden_value]
 * `-l, --visibility TEXT` - Defines who can see the resource, can be set to one of ['private', 'tenant', 'global'] [default: tenant].
+* `--hidden-value` - The secret value is only shown to the user that created the secret and to admins. Use of the secret is allowed according to user roles and the visibility of the secret.
```

Comment:
Let's add the tenant admins as well ("The secret value is only presented to the user who created it, tenant managers and sys-admins") so there's no misunderstandings